rules/xcodeproj.bzl

```diff
@@ -136,9 +139,14 @@ def _xcodeproj_aspect_impl(target, ctx):
         asset_srcs = [f for f in asset_srcs if _is_current_project_file(f)]
         framework_includes = _get_attr_values_for_name(deps, _SrcsInfo, "framework_includes")
         cc_defines = _get_attr_values_for_name(deps, _SrcsInfo, "cc_defines")
+        swift_defines = _get_attr_values_for_name(deps, _SrcsInfo, "swift_defines")
         if CcInfo in target:
             framework_includes.append(target[CcInfo].compilation_context.framework_includes)
             cc_defines.append(target[CcInfo].compilation_context.defines)
+
+        if SwiftInfo in target:
+            swift_defines.append(depset(target[SwiftInfo].direct_defines))
+            swift_defines.append(target[SwiftInfo].transitive_defines)
```

Comment:
Should we also be looking at the `transitive_modulemaps` field in SwiftInfo? I dug in a little bit, but I'm not sure.
The field contains "The transitive module map files that will be passed to Clang using the <code>-fmodule-map-file</code> option."

Reply:
put them into this setting? Or into a separate build setting?

Reply:
it'd go into `OTHER_SWIFT_FLAGS`, if its needed

Reply:
i will create a separate ticket to track this

Reply:
separate ticket created